playbook/app/pb_kits/playbook/pb_currency/_currency.scss

```diff
@@ -24,6 +24,29 @@
   [class^=pb_currency_wrapper] {
     display: flex;
 
+    &[class*=_deemphasized] [class^=pb_body_kit][class*=_light] {
+      color: $text_lt_default;
+    }
+
+    &[class*=_deemphasized] [class^=pb_title_kit][class*=_4] {
+      color: $text_lt_default;
+      font-weight: 400;
```

Comment:
Found there is a `$regular` font-weight variable with a value of 400 defined in `_typography.scss`.